modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionAssembleFromTask/client-extension/client-extension.yaml

```diff
@@ -0,0 +1,4 @@
+assemble:
+    - fromTask: fooTask
```

Comment:
Should you add a test that actually does `fromTask: bootJar` as well?

Reply:
Do you think we should add it? If it adds additional coverage I think it's a good idea, though don't both tasks ultimately just output files?